### ISSUE TITLE
Continue from #8

### ISSUE THREAD

**Issue by ziqian0925** [ductuantruong/tcm_add#9]
Hi, 

In your paper you mentioned that you trained using ASV spoof 2019 LA dataset. But in #8, you mentioned that you trained 2 separate models for LA (RawBoost --algo 5) and DF (RawBoost --algo 3) evaluation. I'm quite confused on this, could you pls clarify? Thanks.

**Comment by ductuantruong** [comment]
Hi @ziqian0925,

We still only used the ASVspoof 2019LA as training data. However, we trained 2 separate models with two different RawBoost data augmentation setting for evaluating on LA test set (using RawBoost --algo 5) and DF test set (RawBoost --algo 3) evaluation. This practice was previously used in other famous baselines [1,2] so we just followed them for fair comparison

I hope my answer addresses your concern. If you find the work helpful, a star to this repo is much appreciated.

[1] https://github.com/TakHemlata/SSL_Anti-spoofing
[2] https://github.com/ErosRos/conformer-based-classifier-for-anti-spoofing
